src/util/sortBy.js

```diff
@@ -1,6 +1,6 @@
 //@flow
 
-type PluckFn<T> = (T) => any;
+export type PluckFn<T> = (T) => any;
```

Comment:
Probably shouldn't be a part of this PR, but I wonder if we should change the type signature to `(T) => number | string`. I can't really think of a case where trying to compare on something other than numbers or strings is a good idea.

Reply:
I was wondering that too